index.html

```diff
@@ -110,6 +110,28 @@ <h3 class="works_headline">My Recent Works</h3>
           class="works_headline_indicator"
         />
       </div>
+      <!-- CARD WORK PROJECT DETAILS -->
+      <div id="work_project_details" class="work_project_details">
+        
+        <div class="work_project_details_card">
+          <span class="work_detail_close_icon">&times; </span>
+          <img class="work_project_details_img" src="./media/images/projects/youtube.PNG" alt="project screenshot"/>
+            <div class="work_project_details_title_btns"><h5 class="work_project_details_title"></h5>
+              <div class="work_project_details_btn_container btns_title">
+                <button type="button" class="buttons btns_title_live">See Live <img src="./media/Icons/seeliveicon.svg" alt="see live icon"/></button>
+                <button type="button" class="buttons btns_title_github">See Source <img src="./media/Icons/Icon-GitHub.svg" alt="see github icon"/></button>
```

Comment:
- [ ] Please leave more space below buttons on mobile view: 

![image](https://user-images.githubusercontent.com/77038610/151189019-d8fcbdbc-47f1-4d51-9a77-59263aea1459.png)

- [ ] Please check the margins and paddings of the content in Figma and try to apply them in your project 😉